src/lib/form-field/_form-field-legacy-theme.scss

```diff
@@ -56,6 +56,21 @@ $mat-form-field-legacy-dedupe: 0;
   $mat-form-field-legacy-dedupe: $mat-form-field-legacy-dedupe + 0.00001 !global;
 }
 
+// Same as mixin above, but omits the translateZ for printing purposes.
+@mixin _mat-form-field-legacy-label-floating-print($font-scale, $infix-padding, $infix-margin-top) {
+  // We use perspective to fix the text blurriness as described here:
+  // http://www.useragentman.com/blog/2014/05/04/fixing-typography-inside-of-2-d-css-transforms/
+  // This results in a small jitter after the label floats on Firefox, which the
+  // translateZ fixes.
+  transform: translateY(-$infix-margin-top - $infix-padding) scale($font-scale) perspective(100px);
```

Comment:
If you remove the `perspective(100px)` from this rule, I think we can get rid of the whole `-ms-transform` rule below. You'll probably need to move the `+ $mat-form-field-legacy-dedupe` into the `translateY` though.